README.md

```diff
@@ -766,6 +766,111 @@ The `/api/v1/export` endpoint should return the following response:
 Extra labels may be added to all the imported time series by passing `extra_label=name=value` query args.
 For example, `/api/put?extra_label=foo=bar` would add `{foo="bar"}` label to all the ingested metrics.
 
+## How to send data from NewRelic infrastructure agent
+
+VictoriaMetrics accepts data from NewRelic infrastructure agent API at `/api/v1/newrelic/infra/v2/metrics/events/bulk` path.
+NewRelic's infrastructure agent sends so-called [Events](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#event-data)
+which are transformed to the [Prometheus exposition format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format).
+
+By default, NewRelic's infrastructure agent sends next types of the infrastructure events:
+1. SystemSample
+2. ProcessSample
+3. StorageSample
+4. NetworkSample
+5. ContainerSample
+6. InfrastructureEvent
+Those all events processed by VictoriaMetrics and prepare the [Prometheus exposition format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format).
+
+### Sending metrics to VictoriaMetrics
```

Comment:
this shouldn't be a section, as it will be used as an anchor by docs engine. I.e. `#sending-metrics-to-victoriametrics`. This anchor has no connection to the subject. So `http://docs.victoriametrics.com#sending-metrics-to-victoriametrics` will have no notion that this is related to new relic.
PTAL how other doc section for ingestion are formatted.

Reply:
updated